doc/nrf/libraries/bluetooth_services/bt_le_adv_prov.rst

```diff
@@ -0,0 +1,108 @@
+.. _bt_le_adv_prov_readme:
+
+Bluetooth LE advertising providers
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth LE advertising providers subsystem manages advertising and scan response data.
```

Comment:
Maybe `The Bluetooth LE advertising providers subsystem manages advertising data and scan response data.`? I don't know if it's correct when it comes to grammar, but at least that clearly emphasizes that this module manages advertising data and not advertising.

Reply:
Also updated in other places